README.md

```diff
@@ -91,7 +103,11 @@ Define a function called `doSomething`. It should take one argument, called
 
 Your Answer:
 ```js
-// write code here
+function doSomething(thingToDo){
+  var thingToDo = function(){
+    console.log("All the things!");
+  }
```

Comment:
Not quite, since we are passing thingToDo as an argument, we don't need to define it again locally. Additionally, we are looking for you to invoke that function:
```
function doSomething(thingToDo){
  thingToDo();
}
doSomething(function(){
  alert("Yay!");
});
```